modules/executive/components/VoteModal/DefaultView.tsx

```diff
@@ -0,0 +1,350 @@
+import { useBreakpointIndex } from '@theme-ui/match-media';
```

Comment:
extracted from the gigantic vote modal + added some logic for signing the message